azure-template-tox-job.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - bash: |
           set -e
           ls -la /Applications/Xcode*
-          sudo xcode-select --switch /Applications/Xcode_12.2.app/Contents/Developer
+          sudo xcode-select --switch /Applications/Xcode_12.5.1.app/Contents/Developer
```

Comment:
Guessing this is required for the macOS upgrade to work?

Reply:
Yes, XCode `12.2` is no longer available in MacOS 11. I think `12.3` is the earliest version of XCode 12 that's available, and `12.5.1` is the latest version (unless we also want to upgrade to XCode 13).

Reply:
Cool thanks!